CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 
+## [1.5.0]
```

Comment:
In general, @twibiral is the one to update the changelog when it is time for the next release. I'm not sure what the correct procedure is or what he would prefer re: this commit, but I wanted to make sure you were aware of that :) sorry!

Reply:
@chlohal is right, I would prefer if you could remove this @andremeireles. The version number is finalized before a new version is released and added to the CHANGELOG by a script

Reply:
Ah, great.
I think it is now ready to receive the merge.

Reply:
LGTM!